data/hai/hai reveal 6 end.txt

```diff
@@ -0,0 +1,672 @@
+			`	You look from Xilin to Sayari and do your best to look like this isn't totally ruining the lovely breakfast you just had.`
+
+				
+event "mountaintop ascendant"
+	system "Ultima Thule"
+		object
+			sprite star/f0
+			period 10
+		object
+			sprite planet/desert5
+			distance 200
+			period 22.3607
+		object
+			sprite planet/europa-b
+			distance 403.41
+			period 64.056
+		object
+			sprite planet/desert7-b
+			distance 824.9
+			period 187.302
+		object
+			sprite planet/gas1
+			distance 1706.99
+			period 557.553
+			object
+				sprite planet/ice8-b
+				distance 216
+				period 11.1964
+		object "Wormhole Alpha"
+			sprite planet/wormhole
+			distance 4208.63
+			period 2158.5
+			object Mountaintop
+				sprite "planet/station hai eight geocoris"
+				distance 315
+				period 31.1964
+	planet Mountaintop
+		government "Hai"
+		attributes hai "hai tourism"
+		landscape land/station0
+		description `This brand new station is built with wide spacious interiors to accommodate the Hai's preferred geometric development pattern and long-term perspectives. The powerful holoprojection system which projected an 'anomaly' into space to cover the wormhole is now inactive. Now that the Hai are entering public relations with human governments this station has come into its own. A wide concourse facilitates social and business exchanges, as well as parades and demonstrations. The two uppermost decks are given over entirely to the state functions and diplomatic services including the now famed Borealis Chamber, formally referred to as 'Where Mountains Touch The Sky', where the public acknowledgment of the Hai was first given and the two species committed a formal resolution to define a treaty of cooperation.`
+		spaceport `The spaceport is a spacious docking assembly sheltered under broad carapace-like structures which are arrayed around the lower decks like petals. It is designed to look impressive and still has that shiny look and freshly oiled smell of a newly minted construction, though it has only basic facilities and services.`
+		security 1
+	
+mission "Hai Reveal [D02] Cleanup"
+	landing
+	name "Hai Diplomacy"
+	description "Pick up a misplaced group of kidnapped Hai and find out what happened."
+	source "Mountaintop"
+	stopover "Sunracer"
+	destination "Mountaintop"
+	passengers 23
+	blocked "You need <bunks> passenger spaces to pick up the misplaced Hai, make sure you have that and return here."
+	to offer
+		has "Hai Reveal [D01] Cleanup: done"
+	on stopover
+		conversation
+			`Arriving to Sunracer you follow the coded coordinates you were given to avoid drawing unnecessary attention. No one wants any press on this with the Hai's first public appearance only just days past and still dominating the news cycle. Fortunately a glance at the feeds your ship can pick up on the way in seems to indicate that the government response has taken the wind out of the more sensationalist talking heads, or at least the ones with notable followings.`
```

Comment:
I wanted to provide some sort of direct indication that the hysterical populists mentioned waaayy back in part 1 had been somewhat effectively countered. It was just closure for a minor point.

Reply:
Gotcha. I didn't make that connection myself, but it's also been a good while since I read part 1.
```suggestion
			``Arriving on Sunracer, you follow the coded coordinates you were given to avoid drawing unnecessary attention. With the Hai's first public appearance only just days past and still dominating the news cycle, you've been instructed to keep this as low-profile as possible. Fortunately, a glance at the feeds your ship picks up seems to indicate that the government response has taken the wind out of the more sensationalist talking heads - or at least the ones with notable followings.`
```
Just made a couple edits, but threw the last sentence back in. This look good?